### PR TITLE
Add missing CCKS labels to docs

### PR DESCRIPTION
**Pull request by dalees**

### CODE REVIEW

File: source/kubernetes/appendix.rst [hunk 1]
```diff
@@ -105,6 +105,29 @@ Available cluster labels
      - Maximum number of worker nodes to scale out to, if auto scaling is enabled.
 
        This value is required if ``auto_scaling_enabled`` is ``true``.
+   * - ``auto_healing_enabled``
+     - Boolean
+     - ``true``
+     - ``true`` or ``false``
+     - Enable auto-healing on control plane and worker nodes.
+
+       With auto-healing enabled, if nodes become NotReady for an extended duration they will be
```
Comment (Callum027) [inline source/kubernetes/appendix.rst:114]: Don't feel strongly about this one, I personally think this looks nicer though.

```suggestion
       With auto-healing enabled, if nodes become ``NotReady`` for an extended duration they will be
```

File: source/kubernetes/appendix.rst [hunk 2]
```diff
@@ -105,6 +105,29 @@ Available cluster labels
...
+       if 40% are considered unhealthy, preventing some cascading failures.
+
+   * - ``keystone_auth_enabled``
+     - Boolean
+     - ``true``
+     - ``true`` or ``false``
+     - With this option enabled, a deployment will be installed into your cluster allowing the use
+       of Role-Based Access Control with Catalyst Cloud's authentication system.
+
+       For more information see :ref:`k8s-rbac-roles`
```
Comment (Callum027) [inline source/kubernetes/appendix.rst:127]: Add a period to the end of this.

```suggestion
       For more information see :ref:`k8s-rbac-roles`.
```

**Review by Callum027 — COMMENTED** [review]
This renders properly and looks correct.

Two small comments I could think of, nitpicks so if they're not a big deal I'm fine to merge this.

File: source/kubernetes/appendix.rst (hunk 1 above)
Comment (dalees) [reply source/kubernetes/appendix.rst]: Done

File: source/kubernetes/appendix.rst (hunk 2 above)
Comment (dalees) [reply source/kubernetes/appendix.rst]: Done

**Pull request merged**